forgevtt-module.js

```diff
@@ -167,15 +167,18 @@ class ForgeVTT {
                 return reject(`Failed to load texture ${src}`);
             };
         } else {
+            const TextureLoader = (foundry && foundry.canvas && foundry.canvas.TextureLoader)
```

Comment:
question: why not use `isNewerVersion` instead of this condition?

Using `isNewerVersion` feels more Foundry-like way to support multiple Foundry versions

Reply:
I think I'm inclined to agree with using `isNewerVersion`. I'd have a different opinion if optional chaining were on the table.

Though I think I'd prefer if we had a separate class/module that handles this sort of version compatibility mapping for us, rather than scattering it throughout our code. Something like...
```js
class ForgeCompatibilityLayer {
  static get TextureLoader() {
    if (isV13) {
      return foundry.canvas.TextureLoader
    } else if (someOtherCondition) {
      // ...
    }
    return undefined
  }
}
```
IMO the less the rest of the code needs to think about Foundry versions, the better.

Reply:
Optional chaining is definitely on the table here, this code only ever runs in a modern browser (has to support Foundry).